data/json/mutations/mutations.json

```diff
@@ -3954,6 +3954,79 @@
     "movecost_modifier": 0.88,
     "category": [ "BATRACHIAN", "RABBIT", "FELINE", "LUPINE", "SPIDER" ]
   },
+  {
+    "type": "mutation",
+    "id": "ANIMAL_FEET",
+    "name": { "str": "Digitigrade Legs" },
+    "points": -1,
+    "visibility": 2,
+    "ugliness": 2,
+    "description": "The heels of your feet have shifted up off the ground in a rather animalistic fashion.  While *some people* might think this looks cool, it mostly just means you can't carry as much or wear shoes.  At least you can move faster when crouching.",
+    "types": [ "FEET" ],
+    "prereqs": [ "PADDED_FEET" ],
+    "category": [ "LUPINE", "FELINE", "BEAST", "RABBIT" ],
+    "changes_to": [ "RABBIT_FEET" ],
+    "wet_protection": [ { "part": "foot_l", "neutral": 10 }, { "part": "foot_r", "neutral": 10 } ],
+    "restricts_gear": [ "foot_l", "foot_r" ],
+    "destroys_gear": true,
+    "weight_capacity_modifier": 0.8,
+    "triggers": [
+      [
+        {
+          "condition": {
+            "and": [
+              { "u_has_move_mode": "run" },
+              { "not": "u_can_drop_weapon" },
+              { "u_has_any_trait": [ "THRESH_BEAST", "THRESH_FELINE", "THRESH_LUPINE" ] },
+              { "u_has_trait": "PAWS" }
+            ]
+          },
+          "msg_on": { "text": "You drop down on all fours." },
+          "msg_off": { "text": "You rise up to walk on your hind feet." }
+        }
+      ]
+    ],
+    "enchantments": [
+      {
+        "condition": {
+          "and": [
+            { "u_has_move_mode": "crouch" },
+            { "not": "u_can_drop_weapon" },
+            { "u_has_any_trait": [ "THRESH_LUPINE", "THRESH_FELINE", "THRESH_BEAST" ] }
+          ]
+        },
+        "values": [ { "value": "MOVE_COST", "multiply": -0.15 }, { "value": "CARRY_WEIGHT", "multiply": 0.35 } ]
```

Comment:
So this reduces your carry_weight by 65%? Am I reading that right?

Reply:
The fact that you explained that and I still don't understand it, and I have no idea what happens if the character has this mutation but not PAWS (which will be a pretty common situation) suggests to me that you might have overthought this mutation. Players should be able to understand what is happening and how when something this intrinsic to their character is being changed.

Reply:
Then you probably want to remove this line: "At least you can move faster when crouching." from the description.

Reply:
If you don't want to leave it blank, I would suggest clarifying it even a little more - "These are a clumsy match for human arms, but for someone with paws up front, they'd be quite helpful."